packages/sdk/src/actor/actor.ts

```diff
@@ -198,6 +198,16 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
 	 * PUBLIC METHODS
 	 */
 
+	/**
+	 * Destroys the collider.
+	 */
+	public clearCollider(): void {
```

Comment:
Good find! I don't think I've ever needed to remove a collider without removing the whole actor before, but I agree it should be possible.